msg/OrientationAngles.msg

```diff
@@ -1,4 +1,10 @@
-# Orientation, angles in degrees
+# Orientation, angles in radians
+
+# Pitch is defined as rotation around the y axis. Positive pitch means pitching in the direction of +x
 float64 pitch
-float64 yaw
+# Roll is defined as rotation around the x axis. Positive pitch means pitching in the direction of -y
 float64 roll
+
+# Yaw is defined as rotation around the z axis. 0 yaw corresponds with the +x axis.
+# Positive rotation indicates CCW rotation when looking down the +z axis towards the origin.
```

Comment:
Actually, can we define pitch as rotation around -y and yaw as rotation around -z?  That would make everything consistent and right-handed (just using different coordinate axes from the `level_quad` frame), and it's the convention that's typically used for those angles.